combine_assets.php

```diff
@@ -4,12 +4,25 @@
     'js/sensitive_banner_all.js' => [
         'sensitive-banner-static/res/sensitive-banner.js',
         'sensitive-banner-static/res/lightbox.js',
+        'js/lib/function-bind.min.js',
         'js/banner/banner.js',
         'js/banner/banner.config.js',
         'js/banner/banner.encryption.js',
         'js/banner/banner.api.js',
         'js/banner/banner.form.js',
     ],
+    'js/lib/pgp-legacy-libs.js' => [
+        'js/lib/es6-promise.min.js',
+        'js/lib/base64.min.js', # The legacy PGP library is not compatible with the base64 polyfill so we have to include two libraries for base64 encoding :(
```

Comment:
You're right, it is included for `banner.api.js` so it makes more sense to move it to the general lib.